src/wled2.html

```diff
@@ -29,6 +40,37 @@
     oneditprepare: function() {
       $("#node-input-address").change({ currentEffect: this.effect, currentPalette: this.palette }, loadDropdowns);
 
+      $("label").click(ev => {

```

Comment:
Some code comments and longer variable names would be helpful here to explain what's going on. `ev` and `em` aren't descriptive enough, and since there are no comments around it's tough to tell what's happening.